final_prep/mod_zero_hero.rb

```diff
@@ -1,84 +1,131 @@
 #   - Danger levels that are above 50 are too scary for your hero. Any danger level that is above 50 should result in printing the bad_excuse to the terminal
 #   - Anything danger_level that is between 10 and 50 should result in printing the save_the_day string to the terminal
 #   - If the danger_level is below 10, it means it is not worth your time and should result in printing the string "Meh. Hard pass." to the terminal.
-
+  if danger_level > 50
+    puts "#{bad_excuse}"
+  elsif danger_level > 10 && danger_level < 50
+    puts "#{save_the_day}"
+  else danger_level < 10
+    puts "Meh. Hard pass."
+  end
+end
 #Test Cases
 announcement = 'Never fear, the Courageous Curly Bracket is here!'
 excuse = 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.'
 # assess_situation(99, announcement, excuse) > Should print - 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.'
 #assess_situation(21, announcement, excuse) > should print - 'Never fear, the Courageous Curly Bracket is here!'
 #assess_situation(3, announcement, excuse) > should print - "Meh. Hard pass."
-
+def assess_situation(number, announcement, excuse)
+  if number == 99
+    puts "#{excuse}"
+  elsif number == 21
+    puts "#{announcement}"
+  else number == 3
+    puts "Meh. Hard Pass"
+  end
+end
 # Declare a new variable - scary_monster - assigned to an hash with the following key/values
 #   - name (string)
 #   - smell (string)
 #   - weight (integer)
 #   - citiesDestroyed (array)
 #   - luckyNumbers (array)
 #   - address (hash with following key/values: number , street , state, zip)
-
+scary_monster = {"name" => "Christian", "smell" => "good", "weight" => 210, "citiesDestroyed" => ["Tampa", "Raleigh", "Denver"], "luckyNumbers" => [4, 7, 44],
+"address" => {"number" => 200, "street" => "Wallaby Way", "state" => "CO", "zip" => 80202}}
 
 # Create a new class called SuperHero
 # - Your class should have the following DYNAMIC values
-#   - name 
+#   - name
 #   - super_power
-#   - age 
+#   - age
 # - Your class should have the following STATIC values
 #   - arch_nemesis, assigned to "The Syntax Error"
 #   - power_level = 100
-#   - energy_level = 50 
+#   - energy_level = 50
+class SuperHero
+  attr_accessor :name, :super_power, :age, :arch_nemesis, :power_level, :energy_level
+
+  def initialize(name, super_power, age, arch_nemesis, power_level, energy_level)
+    @name = name
+    @super_power = super_power
+    @age = age
+    @arch_nemesis = "The Syntax Error"
+    @power_level = 100
+    @energy_level = 50
+  end
 
 # - Create the following class methods
 #   - say_name, should print the hero's name to the terminal
 #   - maximize_energy, should update the energy_level to 1000
 #   - gain_power, should take an argument of a number and INCREASE the power_level by that number
-
+  def say_name
+    puts "#{name}"
+  end
+  def maximize_energy
+    energy_level = 1000
+  end
+  def gain_power(random_number)
+    power_level + random_number
```

Comment:
Don't forget you want to re-assign the `power_level` variable to the result of `power_level + random_number`